### ISSUE TITLE
Duplicate absMid

### ISSUE THREAD

**Issue by agavrilov** [OpenNTF/dojo-webpack-plugin#232]
I get 'Duplicate absMid' on 2.8.5 version:
```
ERROR in chunk calendar [entry]
Duplicate absMid ("lib/core") for modules "C:\\P\\git\\c8\\Projects\\WebAccess\\node_modules\\diagram-js\\lib\\core\\index.js" and "C:\\P\\git\\c8\\Projects\\WebAccess\\node_modules\\bpmn-js\\lib\\core\\index.js"
Error: Duplicate absMid ("lib/core") for modules "C:\\P\\git\\c8\\Projects\\WebAccess\\node_modules\\diagram-js\\lib\\core\\index.js" and "C:\\P\\git\\c8\\Projects\\WebAccess\\node_modules\\bpmn-js\\lib\\core\\index.js"
    at DojoAMDChunkTemplatePlugin.<anonymous> (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDChunkTemplatePlugin.js:77:11)
    at module.filterAbsMids.module.filterAbsMids.absMid (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDChunkTemplatePlugin.js:83:5)
    at absMids.forEach.absMid (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDModuleFactoryPlugin.js:232:8)
    at Array.forEach (<anonymous>)
    at DojoAMDModuleFactoryPlugin.filterAbsMids (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDModuleFactoryPlugin.js:231:11)
    at SyncHook.eval [as call] (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:1)
    at callHook (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:130:21)
    at callSync (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:137:10)
    at NormalModule.module.filterAbsMids.callback [as filterAbsMids] (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDModuleFactoryPlugin.js:409:4)
    at modules.forEach (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDChunkTemplatePlugin.js:82:35)
    at Array.forEach (<anonymous>)
    at DojoAMDChunkTemplatePlugin.renderAbsMids (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDChunkTemplatePlugin.js:80:11)
    at SyncWaterfallHook.eval [as call] (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:16)
    at SyncWaterfallHook.lazyCompileHook (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\Hook.js:154:20)
    at callHook (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:130:21)
    at callSyncWaterfall (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:140:10)
    at DojoAMDMainTemplatePlugin.dojoRequireExtensions (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDMainTemplatePlugin.js:99:12)
    at SyncWaterfallHook.eval [as call] (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:16)
    at SyncWaterfallHook.lazyCompileHook (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\Hook.js:154:20)
    at callHook (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:130:21)
    at callSyncWaterfall (C:\P\git\c8\Projects\WebAccess\node_modules\webpack-plugin-compat\lib\pluginCompatV4.js:140:10)
    at DojoAMDMainTemplatePlugin.requireExtensions (C:\P\git\c8\Projects\WebAccess\node_modules\dojo-webpack-plugin\lib\DojoAMDMainTemplatePlugin.js:80:10)
    at SyncWaterfallHook.eval [as call] (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:17:16)
    at SyncWaterfallHook.lazyCompileHook (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\Hook.js:154:20)
    at MainTemplate.renderBootstrap (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\MainTemplate.js:406:55)
    at MainTemplate.updateHashForChunk (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\MainTemplate.js:534:27)
    at Compilation.createHash (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compilation.js:2325:14)
    at hooks.optimizeTree.callAsync.err (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compilation.js:1288:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\Hook.js:154:20)
    at Compilation.seal (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compilation.js:1244:27)
    at hooks.make.callAsync.err (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compiler.js:624:17)
    at _done (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:9:1)
    at _err17 (eval at create (C:\P\git\c8\Projects\WebAccess\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:224:22)
    at _addModuleChain (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compilation.js:1095:12)
    at processModuleDependencies.err (C:\P\git\c8\Projects\WebAccess\node_modules\webpack\lib\Compilation.js:1007:9)
```

Webpack - 4.29.6

**Comment by chuckdumont** [comment]
I tried adding diagram-js and bpmn-js to the [sample app](https://github.com/OpenNTF/dojo-webpack-plugin-sample), but I can't reproduce the problem.  I added the lib/core dependency for each package to bootstrap.js as follows:

```javascript
define([
	"dojo/ready",
	"dojo/parser",
	"dijit/layout/TabContainer",
	"js/LazyContentPane",
	"dojo/has!webpack?dojo-webpack-plugin/amd/dojoES6Promise",
	"css!dijit/themes/claro/claro.css",
	"css!theme/sample.less",
	"diagram-js/lib/core",
	"bpmn-js/lib/core"
], function(ready, parser) {
	ready(function() {
		parser.parse();
	});
	return {};
});
```

Could you please make sure that you are actually using v2.8.5 of dojo-webpack-plugin.  Look in node_modules/dojo-webpack-plugin/package.json to confirm.  If you are using the correct version, then could you maybe provide a test case that reproduces the problem?

**Comment by agavrilov** [comment]
Here is the reproducible example:
https://github.com/agavrilov/dojo-webpack-plugin-sample/tree/dojo-webpack-plugin-issue-232
Just run:
npm install
npm run build

**Comment by chuckdumont** [comment]
After editing package.json to update dojo-webpack-plugin to v2.8.5 and webpack to v4.29.6, and adding a dependency for uglifyjs-webpack-plugin, the project builds just fine.

**Comment by agavrilov** [comment]
Probably, you built 'master' branch. I have a reproducible example in 'dojo-webpack-plugin-issue-232' branch. It has dojo-webpack-plugin v2.8.5 and webpack v4.29.6 already.

**Comment by chuckdumont** [comment]
OK, I can reproduce now.  Looking into it.

**Comment by chuckdumont** [comment]
Fixed in release 2.8.6
